src/cdk/overlay/position/flexible-connected-position-strategy.spec.ts

```diff
@@ -767,6 +767,36 @@ describe('FlexibleConnectedPositionStrategy', () => {
 
     });
 
+    describe('with origin set to an SVG element', () => {
+      beforeEach(() => {
```

Comment:
Since it's a single test you don't have to do it inside a `beforeEach`. Also it might be worth tweaking `createBlockElement` so the styling code doesn't have to be repeated.

Reply:
Done